### ISSUE TITLE
Handle first sync event ( sync world )

### ISSUE THREAD

**Issue by Unsigno** [lance-gg/lance#68]
For convenience reasons ( eg. initialize visual scene/world ) should be usefull a firstSyncReceived/worldSync or something similar , maybe with pre/post addition .

**Comment by OpherV** [comment]
You're correct in that practically in all cases the game engine has "nothing to do" before the first full sync. 
but maybe instead of adding to the API we can just make sure that the 'start' for the client game engine only happens after it receives the first full sync.  

Thoughts?

**Comment by Unsigno** [comment]
Was the original purpose, but as you mention some one should be interested on run some "local scene", maybe loading another assets or waiting for connection . So could be diferent things , handle first sync and allow run step before or when first sync .
